db/migrate/20210524130734_create_team_results.rb

```diff
@@ -0,0 +1,15 @@
+class CreateTeamResults < ActiveRecord::Migration[6.0]
+  def change
+    create_table :team_results do |t|
+      t.references :team, foreign_key: true, index: false, null: false
+      t.integer :year, null: false
+      t.integer :rank, null: false
+      t.integer :wins_count, null: false
+      t.integer :loses_count, null: false
```

Comment:
how about adding `default: 0` ?

Reply:
It's good idea. I will apply it, thank you.

Reply:
:done: